UnitedIncome/components
# TypeScript Module Aliasing

### Description
> Please provide a brief description of your changes below.

Re-implements module aliasing into TypeScript. Thanks to @labichn-cof we were able to get `yarn test` working.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Run `yarn test` and `yarn testSingle`, both will pass tests. Run the tests on the `Box.tsx` file as I've added path alaises to it too.
* VSCode should properly resolve the aliasing for you, magic! 

### Comments
> Any additional comments that will aid with the review process.

This will allow us to keep all of our docs intact.
